src/v2/Apps/Auction/Components/AuctionDetails/CascadingEndTimesBanner.tsx

```diff
@@ -5,22 +5,33 @@ import { getENV } from "v2/Utils/getENV"
 
 interface CascadingEndTimesBannerProps {
   cascadingEndTimeIntervalMinutes: number
+  extendedBiddingIntervalMinutes: number | null
 }
 
 export const CascadingEndTimesBanner: React.FC<CascadingEndTimesBannerProps> = ({
   cascadingEndTimeIntervalMinutes,
+  extendedBiddingIntervalMinutes,
 }) => {
   const helpArticleLink = getENV("CASCADING_AUCTION_HELP_ARTICLE_LINK")
 
   const hasLink = !!helpArticleLink
+  function getBannerText(extendedBiddingIntervalMinutes) {
+    let bannerText
+    if (extendedBiddingIntervalMinutes) {
```

Comment:
So if an auction has both extended and cascading on (as they would when extended bidding is enabled), the copy will just reference the extended bidding info? (I think that's fine, just double-checking).

Reply:
That's my understanding from my conversation with the auctions team the other day!